bin/ruby-petstore-faraday.json

```diff
@@ -0,0 +1,5 @@
+{
+  "gemName": "petstore",
```

Comment:
I think `typhoeus` is the default so we need to add `"library": "faraday",` here if I'm not mistaken.

Reply:
I'll take care of it in another PR.